src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -107,12 +108,6 @@ def delete_user(
             status_code=HTTP_404_NOT_FOUND, detail=f"No user found with id {user_id}."
         )
 
-    if not (client.fides_key == ADMIN_UI_ROOT or client.user_id == user.id):
-        raise HTTPException(
-            status_code=HTTP_403_FORBIDDEN,
-            detail="Users can only remove themselves, or be the Admin UI Root User.",
-        )
-
     logger.info("Deleting user with id: '{}'.", user_id)
```

Comment:
Now we're loosening the rules for this, it'd be useful to log which user performed this action, consider "User with id {} deleted by user with id {}".